.changeset/kind-crabs-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Add static loading and running icons, and icons for Google services included docs, forms, slides, sheets, and drive.
```

Comment:
`Flight Icons`

- Added "static" version of the animated icons:

    - `loading-static`
    - `running-static`

- Added new set of icons for Google services:

    - `google-docs`
    - `google-forms`
    - `google-slides`
    - `google-sheets`
    - `google-drive`